api/optimeet/optimeet/settings.py

```diff
@@ -0,0 +1,123 @@
+"""
+Django settings for optimeet project.
+
+Generated by 'django-admin startproject' using Django 4.2.6.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-h_*173ykib^8x0m6@elbqelzysfden)m#m1%7==btv%9i@9qr('
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
We'll also want to setup this `DEBUG` variable as an environment variable, since we'll want it to be `True` in our local systems but change it to `False` once we deploy it to a server.

Reply:
Done